sections/intro.tex

```diff
@@ -1,6 +1,290 @@
 \section{Introduction}\label{sec:intro}
 
-This introduction will briefly summarise the literature review for studying a
-patient corpus via clustering. Following this, a condensed data analysis is
-presented highlighting the main conclusions of the clustering and the overall
-benefits compared with traditional condition-treatment segmentation.
+Population health research is becoming increasingly based on data-driven methods
+(as opposed to those designed solely by clinical experts) for patient-centred
+care through the advent of accessible software and a relative abundance of
+electronic data. A vital part of such research is to better understand the
+healthcare needs and behaviours of a population, and it can be beneficial to
+find an appropriate segmentation of that population; such a segmentation allows
+for finer-grained analysis of groups in the population that share some form of
+homogeneity. One commonly used method for such patient-centred analysis is that
+of patient flow and their interaction with the healthcare system.
+
+However, this process relies heavily on detailed data --- about both the system
+and the population within that system --- which may limit research where
+sophisticated data pipelines are not yet in place. This work demonstrates how
+this issue may be overcome using administrative, spell-level hospital data to
+build a patient clustering that feeds into a multi-class queuing model.
+Specifically, this work examines patient records from the National Health
+Service (NHS) Wales Cwm Taf Morgannwg University Health Board (UHB) that present
+chronic obstructive pulmonary disease (COPD). COPD is of particular interest to
+Cwm Taf Morgannwg UHB as the condition is known to often present as a
+comorbidity in patients~\cite{Houben2019} and it was found that they had the
+highest prevalence of the condition across all the Welsh health boards in an
+internal report by NHS Wales.
+
+% TODO No reference for graph sent by Kendal but is this alright?
+
+% TODO Image of process: administrative data -> extract service parameters ->
+%                        validate parameters with Wasserstein distance ->
+%                        continue with queuing model as normal
+
+The remainder of the paper is structured as follows: Section~\ref{sec:intro}
+provides a literature review, and an overview of the data and its clustering;
+Section~\ref{sec:model} describes the queuing model used and the estimation of
+its parameters; Section~\ref{sec:scenarios} presents a number of what-if
+scenarios with insight provided by the model parameterisation and the
+clustering; Section~\ref{sec:conclusion} concludes the paper. Although the data
+is confidential and may not be published, the source code used in this paper is
+available online at~\url{https://github.com/daffidwilde/copd-paper}.
```

Comment:
Can we generate a small synthetic data set as an example? We could archive this and include a notebook in the GitHub repo?

(Not suggesting we do something too clever to generate the synthetic data, just some random numbers will do to showcase the approach.)

Reply:
Yeah, sure thing. I could do a kernel density fit on the real LOS data and sample for each cluster?

Reply:
Something like that yup :+1:

Reply:
That's done now and is archived at https://doi.org/10.5281/zenodo.3908167 👍